src/bundle/ui-dev/src/modules/universal-discovery/services/universal.discovery.service.js

```diff
@@ -130,11 +130,14 @@ export const loadAccordionData = (
         .catch(showErrorNotification);
 };
 
-export const findLocationsBySearchQuery = ({ token, siteaccess, query, limit = QUERY_LIMIT, offset = 0 }, callback) => {
+export const findLocationsBySearchQuery = ({ token, siteaccess, query, limit = QUERY_LIMIT, offset = 0, languageCode = null }, callback) => {
+    const useAlwaysAvailable = true;
```

Comment:
I'm no familiar with this option, is this something I would like to change to false in some cases?

Reply:
@dew326  I don't think it should be changed for false while searching in UDW, its hard-coded to true in main "Search" module as well.

Reply:
_useAlwaysAvailable defaults to true to avoid exceptions on missing translations_